editor/editor_file_system.cpp

```diff
@@ -1714,105 +1714,115 @@ HashSet<StringName> EditorFileSystem::_get_scene_groups(const String &p_path) {
 
 void EditorFileSystem::update_file(const String &p_file) {
 	ERR_FAIL_COND(p_file.is_empty());
-	EditorFileSystemDirectory *fs = nullptr;
-	int cpos = -1;
+	Vector<String> scripts;
+	scripts.push_back(p_file);
+	update_files(&scripts);
+}
 
-	if (!_find_file(p_file, &fs, cpos)) {
-		if (!fs) {
-			return;
+void EditorFileSystem::update_files(const Vector<String> *p_script_paths) {
+	for (int idx_file = 0; idx_file < p_script_paths->size(); idx_file++) {
+		String file = (*p_script_paths)[idx_file];
```

Comment:
```suggestion
void EditorFileSystem::update_files(const Vector<String> &p_script_paths) {
	for (const String &file : p_script_paths) {
```
Prefer for-each syntax when appropriate